app/models/user.rb

```diff
@@ -1,6 +1,10 @@
 class User < ActiveRecord::Base
   has_many :pieces
   has_many :games, :through => :pieces
```

Comment:
may want to remove this later since it's no longer necessary. didn't want to step on any toes!